2_process.R

```diff
@@ -326,6 +327,30 @@ p2_targets_list <- list(
       )}
   ),
   
+  # Combine NLCD and FORESCE
+  ## Cat
+  tar_target(
+    p2_all_lulc_data_cat,
```

Comment:
The column names for land cover classes should have CAT or TOT so they can be distinguished from each other when they're eventually combined into one table. To match the NHD format: `CAT_prop_lcClass_#` 

I'm not sure where the best place to make that edit would be. Maybe better to add when they are combined instead of having to change something we've already reviewed?